packages/platforms/react-native/android/src/main/java/com/bugsnag/android/performance/NativeBugsnagPerformanceImpl.java

```diff
@@ -119,6 +126,12 @@ public WritableMap getNativeConfiguration() {
       return null;
     }
 
+    if (!spanCleanupTaskScheduled) {
+      ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+      scheduler.scheduleWithFixedDelay(this::discardLongRunningSpans, 1, 1, TimeUnit.HOURS);
```

Comment:
Could this perhaps be replaced with a main-thread `Handler` so that we don't require a whole cleanup thread?

Reply:
Updated 👍🏾